### PR TITLE
Fix typo in Ordinals.pm from PR #14074

### PR DESCRIPTION
**Pull request by vdukhovni**
Commit 50ccc176da8644be079eccc4523c261e34f7b293 (PR #14074) broke reporting of duplicate prototypes due to an overlooked typo.

### CODE REVIEW

**Review by levitte — APPROVED** [review]

**Review by hlandau — APPROVED** [review]

**Review by DDvO — APPROVED** [review]
LGTM

**Comment by vdukhovni** [comment]
Who's on the hook for the 3.0 and 3.1 branch "backports" (cherry-pick)? Are additional PRs required?

**Comment by t8m** [comment]
> Who's on the hook for the 3.0 and 3.1 branch "backports" (cherry-pick)? Are additional PRs required?

Not required if the commit cherry-picks cleanly.

**Comment by vdukhovni** [comment]
Do merges here, or upstream somewhere?

**Review by paulidale — APPROVED** [review]

**Comment by vdukhovni** [comment]
Do merges happen in this repo, or in `https://github.openssl.org/openssl/openssl`?

**Comment by paulidale** [comment]
In https://github.openssl.org/openssl/openssl, not here.  It is automatically mirrored back.

**Comment by vdukhovni** [comment]
> In https://github.openssl.org/openssl/openssl, not here. It is automatically mirrored back.

I'll leave it to the team then. I don't think I have write access there at present (and been inactive too long to recall the process).

**Comment by levitte** [comment]
I'lll merge it now

**Comment by levitte** [comment]
Merged

master:
77c8d6d703efd1d935b5c2603fd31f4b15b1214d Fix typo in Ordinals.pm from PR #14074

3.1:
b767b00c81ad0eadca62464050db374fa5712600 Fix typo in Ordinals.pm from PR #14074

3.0:
98d7c6e05f44fbcde79561ea49e5aa5890076935 Fix typo in Ordinals.pm from PR #14074

**Comment by beldmit** [comment]
Is it really merged? The repo is frozen...

**Comment by levitte** [comment]
> Is it really merged? The repo is frozen...

Well, it's always frozen for all but one...  that "one" happens to be me this time.

**Pull request closed without merging**
